src/components/Squeak/components/auth/SignIn.tsx

```diff
@@ -21,7 +21,9 @@ export const SignIn: React.FC<SignInProps> = ({ buttonText = 'Login', onSubmit,
             password: values.password,
         })
 
-        if (user?.error) {
+        if (!user) {
+            setMessage && setMessage('Invalid email or password')
```

Comment:
Is this a valid error message? If the email or password is incorrect, it will move to the next block and provide the correct error message returned from the server. I like what you're doing in `SignUp.tsx`: 

> There was an error signing up. Please try again.

Reply:
To be honest, this was GitHub co-pilot so I take no responsibility 😅 Agreed, I'll update that error message